src/modules/letter-spacing.js

```diff
@@ -0,0 +1,25 @@
+const applyRules = require("../util/applyRules");
+
+const defaultNames = {
+  lts: "lts",
+  "lts-n": "lts-n"
+};
+
+const getRules = (names, cfg) => ({
+  lts: { name: names["lts"], key: "letter-spacing", value: cfg.values },
+  "lts-n": { name: names["lts-n"], key: "letter-spacing", value: "normal" }
```

Comment:
@sylvesteraswin don't know why emmet cheatsheet uses `lets-n`, seems like a typo as the conventions should be `lts:n` and it doesn't seem to collide with anything else

Reply:
I will use the `:` format.